tests/test_data/test_datasets/test_sot_test_dataset.py

```diff
@@ -63,6 +63,42 @@ def test_sot_ope_evaluation():
     assert eval_results['precision'] == 50.0
 
 
+def test_sot_vot_evaluation():
+    dataset_class = DATASETS.get('VOTDataset')
+    dataset = dataset_class(
+        ann_file=osp.join(LASOT_ANN_PATH, 'lasot_test_dummy.json'),
+        pipeline=[])
+
+    for _, img_ann in dataset.coco.anns.items():
+        x, y, w, h = img_ann['bbox']
+        img_ann['bbox'] = [x, y, x + w, y, x + w, y + h, x, y + h]
+
+    results = []
+    for video_name in ['airplane-1', 'airplane-2']:
+        results.extend(
+            mmcv.list_from_file(
+                osp.join(LASOT_ANN_PATH, video_name, 'vot_track_results.txt')))
+    track_bboxes = []
+    for result in results:
+        result = result.split(',')
+        if len(result) == 1:
+            track_bboxes.append([float(result[0]), 0.])
```

Comment:
change `[float(result[0]), 0.]` to a ndarray, since the evalution code take ndarray as input